packages/components/src/QualityBar/QualityBar.component.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { EnrichedQualityType, QualityBarPercentages, QualityCommonProps } from './QualityBar.types';
+import { QualityBarRatioBars } from './QualityBarRatioBars.component';
+import { SplitQualityBar } from './SplitQualityBar.component';
+
+export type QualityBarProps = QualityCommonProps & {
+	disabled?: boolean;
```

Comment:
could this be part of QualityCommonProps ?

Reply:
Could be part of it, but it would required to change other stuff regarding to the typing.
It's used for instance in `getQualityPercentagesRounded` with a `Required` so the property is meant to be present as the others.

Otherwise, it looks more a props rather that a quality attribute to me